hudi-metrics-common/src/main/java/org/apache/hudi/metrics/config/HoodieMetricsConfig.java

```diff
@@ -46,6 +52,11 @@
 
   public static final String METRIC_PREFIX = "hoodie.metrics";
 
+  public static final ConfigProperty<String> TBL_NAME = ConfigProperty
```

Comment:
we don't want to duplicate the configs.can we rely on HoodieTableConfig only for table name.